docs/release-notes/RELEASE-NOTES-0.10.0.md

```diff
@@ -4,6 +4,7 @@
 It:
 * Improves chain handling by storing accumulated difficulty in the db rather than recomputing it. This impacts the persisted DB.
 * Enable user configuration of HTTP API acceptors pool
+* Restructure the transaction root hash in the block header for stricter validation. This impacts consensus and the persisted DB.
```

Comment:
`git diff v0.9.0 origin/master -- apps/aecore/include/blocks.hrl` returns empty so it looks like this PR has the honour of being the first to break consensus.
* Could you please bump block PROTOCOL_VERSION ?
* And could you also bump GENESIS_VERSION - as so far no support for hard fork without chain restart ?

Ref https://github.com/aeternity/epoch/blob/4e96464f8be73205919e68e237433b0e3df88c8c/apps/aecore/include/blocks.hrl#L3-L5